example/test.ts

```diff
@@ -1,5 +1,12 @@
-
-import {from_hex, to_hex, decrypt_and_verify, gen_signing_key, encrypt_and_sign, public_key_from_secret, constant_time_eq } from "x25519-chacha20poly1305";
+import {
+  to_hex,
+  from_hex,
+  gen_signing_key,
+  public_key_from_secret,
+  encrypt_and_sign,
+  decrypt_and_verify,
+  constant_time_eq,
+} from "../pkg/x25519_chacha20poly1305";
```

Comment:
not sure what changed here: I ran the same command that's in the makefile: 



`wasm-pack build --target web`

it's output is now in the `pkg` dir. There something I might have borked?

Reply:
yours looks fine to me, just linting and pointing to the local build, which is what we want